content/blog/2016/2016-01-10-beautiful-jenkins-dashboard.adoc

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title: "A beautiful Jenkins dashboard"
+author: kj187
+---
+
+
+Jenkins offers a handful of third party dashboards, but none of them is really beautiful and flexible from my point of view.
```

Comment:
Makes sense to justify your statement. E.g. reference main requirements, explain why the current solutions do not work.